src/super_gradients/training/utils/callbacks/all_callbacks.py

```diff
@@ -0,0 +1,58 @@
+from super_gradients.training.utils.callbacks.callbacks import DeciLabUploadCallback, LRCallbackBase, LRSchedulerCallback, MetricsUpdateCallback, \
+    ModelConversionCheckCallback, YoloXTrainingStageSwitchCallback, StepLRCallback, PolyLRCallback,\
+    CosineLRCallback, ExponentialLRCallback, FunctionLRCallback, WarmupLRCallback
+
+from super_gradients.training.utils.early_stopping import EarlyStop
+from super_gradients.training.datasets.datasets_utils import DetectionMultiscalePrePredictionCallback
+
+
+class CallbackNames:
+    """Static class holding all the supported callback names"""""
+    DECI_LAB_UPLOAD = 'DeciLabUploadCallback'
+    LR_CALLBACK_BASE = 'LRCallbackBase'
+    LR_SCHEDULER = 'LRSchedulerCallback'
+    METRICS_UPDATE = 'MetricsUpdateCallback'
+    MODEL_CONVERSION_CHECK = 'ModelConversionCheckCallback'
+    EARLY_STOP = 'EarlyStop'
+    DETECTION_MULTISCALE_PREPREDICTION = 'DetectionMultiscalePrePredictionCallback'
+    YOLOX_TRAINING_STAGE_SWITCH = 'YoloXTrainingStageSwitchCallback'
+
+
+CALLBACKS = {
+    CallbackNames.DECI_LAB_UPLOAD: DeciLabUploadCallback,
+    CallbackNames.LR_CALLBACK_BASE: LRCallbackBase,
+    CallbackNames.LR_SCHEDULER: LRSchedulerCallback,
+    CallbackNames.METRICS_UPDATE: MetricsUpdateCallback,
+    CallbackNames.MODEL_CONVERSION_CHECK: ModelConversionCheckCallback,
+    CallbackNames.EARLY_STOP: EarlyStop,
+    CallbackNames.DETECTION_MULTISCALE_PREPREDICTION: DetectionMultiscalePrePredictionCallback,
+    CallbackNames.YOLOX_TRAINING_STAGE_SWITCH: YoloXTrainingStageSwitchCallback
+}
+
+
+# DICT FOR LEGACY LR HARD-CODED REGIMES, WILL BE DELETED IN THE FUTURE
+class LRScheduler:
```

Comment:
Not consistent with your logic of adding the names suffix, can be confusing.

Reply:
Also changed in e53d29780f25ec1cd250d6a50f3a2f1eb0ab2a9a